src/java/org/apache/cassandra/db/memtable/SkipListMemtable.java

```diff
@@ -222,6 +220,24 @@ public Partition getPartition(DecoratedKey key)
         return partitions.get(key);
     }
 
+    @Override
+    public DecoratedKey minPartitionKey()
+    {
+        Map.Entry<PartitionPosition, AtomicBTreePartition> entry = partitions.firstEntry();
```

Comment:
The map offers `firstKey()/lastKey()` as well.

Reply:
I know, but they return `PartitionPosition` and that does not offer `DecoratedKey`.